File: problems/Meeting Rooms/canAttendMeetings.cs
public class Solution {
    public bool CanAttendMeetings(int[][] intervals) {
        Array.Sort(intervals, (a, b) => a[0] - b[0]);

        for (int idx = 1; intervals.Length > idx; ++idx) {
            if (intervals[idx - 1][1] > intervals[idx][0]) {
                return false;
            }
        }

        return true;
    }
}

File: problems/Meeting Rooms/canAttendMeetings.java
class Solution {
    public boolean canAttendMeetings(int[][] intervals) {
        Arrays.sort(intervals, (a,b)->a[0]-b[0]);

        for (int idx = 1; intervals.length > idx; ++idx) {
            if (intervals[idx - 1][1] > intervals[idx][0]) {
                return false;
            }
        }

        return true;
    }
}
